.eslintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    parserOptions: {
+      ecmaVersion: 2019,
+      sourceType: "module",
+    },
+    env: {
+      browser: true,
+      es6: true,
+    },
+    extends: ["airbnb-base"],
+    plugins: ["jest", "svelte3"],
+    rules: {
+      "import/prefer-default-export": "off",
+      "import/no-extraneous-dependencies": ["error", { devDependencies: true }],
+      "no-param-reassign": [
+        "error",
+        { props: true, ignorePropertyModificationsFor: ["draft"] },
+      ],
+    },
+    overrides: [
+      {
+        files: ["**/*.svelte"],
+        processor: "svelte3/svelte3",
+        rules: {
+          "import/first": "off",
+          "import/no-mutable-exports": "off",
+          "prefer-const": "off",
+        },
+      },
+    ],
+  };
```

Comment:
I could help configure Prettier, if you'd like.

Reply:
Sure. I had copied a bunch of stuff from various repositories. Thought I had included prettier here but guess not!

Reply:
https://github.com/graph-paper-org/app-template/pull/10